pkg/systemd/services/service-details.jsx

```diff
@@ -224,6 +285,31 @@ export class ServiceDetails extends React.Component {
         return true;
     }
 
+    doMemoryCurrentPolling() {
+        systemd_client.call(this.props.unit.path,
+                            "org.freedesktop.DBus.Properties", "Get",
+                            [cockpit.format("org.freedesktop.systemd1.$0", this.unitTypeCapitalized), 'MemoryCurrent'])
```

Comment:
(copying from previous review round, still relevant) cockpit.format is a bit overkill for this -- simple + will do.